code/drasil-example/Drasil/SSP/Unitals.hs

```diff
@@ -408,8 +408,7 @@ effectiveStress = uc' "sigma'" (cn' "effective stress")
 
 effNormStress = uc' "sigmaN'" (cn' "effective normal stress")
   ("the normal stress in a soil mass that is effective in causing volume " ++
-   "changes and mobilizes the shear strength arising from friction; " ++ 
-   "represents the average stress carried by the soil skeleton")
+   "changes; represents the average stress carried by the soil skeleton")
```

Comment:
@samm82, I'm sorry to do this, but can you change it from "average stress" to "average normal stress"?

Reply:
Sure thing!